src/app/beer_garden/db/mongo/pruner.py

```diff
@@ -30,7 +32,17 @@ def add_task(
             }
         )
 
+    def cancel_in_progress(self):
```

Comment:
Since this deals in both "IN_PROGRESS" and "CREATED" requests, let's rename this to `cancel_oustanding_requests`.  Similarly, the variable you have named `in_progress` below would become `outstanding_requests`.

Also, let's add a basic docstring describing the method.